README.md

```diff
@@ -41,7 +41,7 @@ apt-get install nodejs
 apt-get install npm
 npm install -g bower
 ln -s /usr/bin/nodejs /usr/bin/node
-pip install django
+pip install https://www.djangoproject.com/download/1.7.8/tarball/
```

Comment:
Sorry, I had encountered that trouble but forgot to modify readme.
pip install django==1.7 is enough